Common/Securities/Cash.cs

```diff
@@ -282,6 +282,14 @@ public void SetAmount(decimal amount)
                 }
             }
 
+            // Special case; even if we don't have USDC-USD pair, still allow USDC cash because 1-1 with USD
+            if (Symbol == "USDC")
+            {
+                ConversionRateSecurity = null;
+                ConversionRate = 1.0m;
```

Comment:
I think this wouldn't work for for users with non-USD account currency.

Shouldn't this case already be handled here?
https://github.com/QuantConnect/Lean/blob/master/Common/Securities/Cash.cs#L238-L247

Reply:
We should probably cover other [stablecoins](https://en.wikipedia.org/wiki/Stablecoin). E.g.:
- `USD`: `USDC`, `USDT`
- `EUR`: `EURS`
- `GBP`: `BGBP`

Reply:
Another option here: https://github.com/QuantConnect/Lean/issues/2060

Reply:
Regarding Stefano's linking to #2060, I've heard that the Bellman-Ford algorithm can be used to solve for the optimal currency conversion path (with the possibility of taking into consideration loss or gain of account equity because of non-efficient conversion rates between pairs) from point `A -> [b, [c, [d, [...]]]] -> Ω` 

This might be an avenue we might want to explore if we decide to implement a fix using that issue's description as the solution

Reply:
As Stefano noted the bigger underlying issue here is https://github.com/QuantConnect/Lean/issues/2060. We decided to add this special case handling for stable coins which don't have any real trading pair and for which #2060 wouldn't work correctly because of it. It will work only when the account currency matches too, same as the rest of the supported cases really until #2060 is fixed. Think this last part is still missing though, what about different account currencies @C-SELLERS? We should add the account currency dimension to the tests 👍